app/src/main/kotlin/co/netguru/baby/monitor/client/feature/client/home/SendFirebaseTokenUseCase.kt

```diff
@@ -3,19 +3,16 @@ package co.netguru.baby.monitor.client.feature.client.home
 import co.netguru.baby.monitor.client.feature.communication.websocket.Message
 import co.netguru.baby.monitor.client.feature.communication.websocket.RxWebSocketClient
 import co.netguru.baby.monitor.client.feature.firebasenotification.FirebaseInstanceManager
-import com.google.gson.Gson
 import io.reactivex.Completable
 import javax.inject.Inject
 
 class SendFirebaseTokenUseCase @Inject constructor(
-    private val firebaseInstanceManager: FirebaseInstanceManager,
-    private val gson: Gson
+    private val firebaseInstanceManager: FirebaseInstanceManager
 ) {
 
     fun sendFirebaseToken(client: RxWebSocketClient): Completable =
         firebaseInstanceManager.getFirebaseToken()
             .flatMapCompletable { token ->
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/b1aa35793dca6d7714e5534d6167592f0bbe6a99/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/client/home/SendFirebaseTokenUseCase.kt#L15"></span>Unexpected indentation (19) (it should be 16)</td>
    </tr>
  </tbody>
</table>